content/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -83,45 +62,187 @@ signature:
-the tests. Creating real infrastructure in
-tests verifies the described behavior of Terraform Plugins in real world use
-cases against the actual APIs, and verifies both local state and remote values
-match. Acceptance tests require a network connection and often require
-credentials to access an account for the given API.
+Enables developers to use the following to run acceptance tests:
+
+```shell
+make testacc
+```
 
-~> Note: When developing or testing Terraform plugins, we highly recommend
-running acceptance tests with an account dedicated to testing. This ensures no
-infrastructure is created or destroyed in error during development or validation
-of any Provider Resources in any environment that cannot be completely and
-safely destroyed.
+### Running Tests with GitHub Actions
+
+If using [GitHub](https://github.com/), run acceptance testing via [GitHub Actions](https://github.com/features/actions). Other continuous integration runners, while not exhaustively documented, are also supported.
+
+Create a [GitHub Actions workflow](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) file, such as `.github/workflows/test.yaml`, that:
+
+- Runs when pull requests are submitted (or on [other events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows) as appropriate)
+- Uses [`actions/checkout`](https://github.com/actions/checkout) to checkout the provider codebase
+- Uses [`actions/setup-go`](https://github.com/actions/setup-go) to install Go
+- Uses [`hashicorp/setup-terraform`](https://github.com/hashicorp/setup-terraform) to install Terraform CLI
+- Runs the `go test` command with the appropriate environment variables and flags
+
+The following example workflow runs acceptance testing for the provider using the latest patch versions of Go 1.17 and Terraform CLI 1.1:
+
+```yaml
+name: Terraform Provider Tests
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/test.yaml'
+      - '**.go'
+
+permissions:
+  # Permission for checking out code
+  contents: read
+
+jobs:
+  acceptance:
+    name: Acceptance Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: '1.1.*'
+          terraform_wrapper: false
+      - run: go test -v -cover ./...
+        env:
+          TF_ACC: '1'
+  unit:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+      - run: go test -v -cover ./...
+```
+
+Use the [`matrix`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) strategy for more advanced configuration, such as running acceptance testing against multiple Terraform CLI versions.
```

Comment:
Maybe move this to above the examples? Just so it doesn't get lost.

Reply:
I'll move it up. 👍 

```suggestion
```